src/entries/popup/pages/settings/walletsAndKeys/walletWipePrompt.tsx

```diff
@@ -22,25 +18,18 @@ import * as wallet from '../../../handlers/wallet';
 const t = (s: string) =>
   i18n.t(s, { scope: 'settings.privacy_and_security.wallets_and_keys' });
 
+async function handleWipeWallet() {
```

Comment:
could maybe inline this now bc it's way smaller now with your refactor. idk what your preference is.

Reply:
I like creating functions outside the component when possible, so it won't be recreated on every rerender, but it's not a huge performance gain here I'd go with whatever you think is more readable, up to you